AlphaWallet/Transfer/ViewControllers/SendViewController.swift

```diff
@@ -298,23 +298,36 @@ extension SendViewController: QRCodeReaderDelegate {
         reader.dismiss(animated: true) { [weak self] in
             self?.activateAmountView()
         }
+        guard let result = QRURLParser.from(string: result) else { return }
+        guard checkEIP861Requirements(result: result) else { return }
+    }
 
-        guard let result = QRURLParser.from(string: result) else {
-            return
+    func checkEIP861Requirements(result: ParserResult) -> Bool {
```

Comment:
Would be good to rename `checkEIP861Requirements()` because it not only checks, it fills the textboxes according to the URL data for both EIP861 and a raw wallet address string (which isn't EIP861).

Reply:
Done